packages/kerosene-test/readme.md

```diff
@@ -0,0 +1,51 @@
+# 🔥 Kerosene-test
+
+`@kablamo/kerosene-test` is not yet available on npm.
+
+## Available Functions
+
+### React
+
+#### `createStubComponent(displayName, functional?, getRenderProps?, transformRenderProps?)`
+
+Creates a StubComponent for unit testing.
+
+#### `createStubContext(displayName, getter?)`
+
+Creates a stub React Context for unit testing.
+
+### Stubs
+
+#### `createStubStyles(classNames, values?)`
+
+Returns an object mapping local classNames to faked global equivalents along with any exported values.
+
+#### `stubProperties(target, properties)`
+
+Stubs `properties` on the `target` object, and returns a function which restores them to their original values.
+
+## Available TypeScript Types
+
+### Jest
```

Comment:
could these perhaps be used as less-specific things? I see we have `JestMock` and `SinonMock`... could we make it so we can use it maybe like this:

`import Mock from '{...}/jest/mock'`
or
`import Mock from '{...}/sinon/mock`

Reply:
Since everything is exported at `src/index.ts` in a single module namespace, not in a straightforward manner.